tests/integration/nvidia_tuned/test_prepare_nvidia_profiles.py

```
@@ -120,6 +130,25 @@ def verify_tuned_version(runner: DockerTestRunner, base_image: str):
         f"tuned version {major}.{minor} is less than required {required_major}.{required_minor} for {base_image}"
 
 
+def expected_profiles_dir(runner: DockerTestRunner) -> str:
+    """Return the dir tuned reads profiles from, based on the container's version.
+
+    tuned >= 2.23.0 -> /etc/tuned/profiles ; older -> /etc/tuned.
+    Mirrors resolve_tuned_profiles_dir in tuned/skyhook_dir/utils.sh.
+    """
+    if runner.container is None:
+        raise RuntimeError("Container not initialized.")
```

Ruff (0.15.20)

[warning] Avoid specifying long messages outside the exception class

(TRY003)